umbrella/Cargo.toml

```diff
@@ -576,6 +577,7 @@ runtime = [
 	"sp-crypto-hashing-proc-macro",
 	"sp-debug-derive",
 	"sp-externalities",
+	"sp-facade-apis/decl-runtime-apis",
```

Comment:
I wonder whether this should be in the "runtime" feature or else in "runtime_full" or kept entirely separate.